from_cpython/Modules/signalmodule.c

```diff
@@ -893,7 +893,7 @@ int
 PyErr_CheckSignals(void)
 {
     // Pyston change:
-    Py_FatalError("TODO");
+    printf("Warning: PyErr_CheckSignals is not yet implemented!\n");
```

Comment:
I think it currently doesn't really matter if we don't do anything in this function, since the only place that can actually set a signal will Py_FatalError().  Can we keep the Py_FatalError() call here too, but move it below the is_tripped check?  that way it doesn't have to be a warning and it will fail more loudly if somehow we do end up setting a signal that should get checked.